CmpsBot/Configuration/EventHandler.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Discord.Commands;
+using Discord.WebSocket;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CmpsBot.Configuration
+{
+    public class EventHandler
+    {
+        private readonly DiscordSocketClient discord;
+
+        public EventHandler(DiscordSocketClient discord)
+        {
+            this.discord = discord;
+            this.discord.UserJoined += NewMemberPrompt;
+        }
+
+        private async Task NewMemberPrompt(SocketGuildUser user)
+        {
+            var guild = user.Guild;
+            var channel = guild.DefaultChannel;
```

Comment:
This will probably end up changing this so that it is configurable via appsettings but I'm fine with this for now.

Reply:
I thought about refactoring it further, but just piggybacked off of the Command Handler's design.

Reply:
Thats fine. If we need to specify a channel via some configuration setting, its an easy enough change.